src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
+        System.out.println("Figure: circle, radius: " + radius + " color: " + Color.WHITE);

```

Comment:
The color should not be hardcoded as 'Color.WHITE'. It should use the color provided in the constructor. Replace 'Color.WHITE' with 'getColor()' to use the color set for the Circle instance.